include/deal.II/multigrid/mg_transfer.h

```diff
@@ -417,6 +417,14 @@ class MGLevelGlobalTransfer : public MGTransferBase<VectorType>
    */
   SmartPointer<const MGConstrainedDoFs, MGLevelGlobalTransfer<VectorType>>
     mg_constrained_dofs;
+
+private:
+  /**
+   * This function is called to make sure that build() has been invoked.
+   */
+  template <int dim, int spacedim>
+  inline DEAL_II_ALWAYS_INLINE void
+  assert_built(const DoFHandler<dim, spacedim> &dof_handler) const;
```

Comment:
Yes, this looks correct to me. Except the `inline DEAL_II_ALWAYS_INLINE` is probably not necessary here because the compiler doesn't know at this point what to inline anyway.

Reply:
I would also remove the always inline statement as we primarily use it as a code optimization vehicle in deal.II.